ourpackage/consultado.py

```diff
@@ -0,0 +1,75 @@
+from colorama import Fore
+import os
+
+def kirby():
+    while True:
+        os.system("cls")
+        print(Fore.GREEN + "    SOMETHING ABOUT ME")
+        print(Fore.RESET + "\n 1. Personal information")
+        print(" 2. My Favorite Quotes")
+        print(" 3. Motivational Story")
+        print(Fore.RED + "\n                    0 - Exit")
```

Comment:
I will add my changes/comment. - Barcelos

Reply:
You have done it.